<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>701</width>
      <height>181</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="PlotWidget" name="gyrox" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>221</width>
       <height>151</height>
      </rect>
     </property>
     <widget class="QLabel" name="gyrox_label">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>0</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
    </widget>
    <widget class="PlotWidget" name="gyroy" native="true">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>221</width>
       <height>151</height>
      </rect>
     </property>
     <widget class="QLabel" name="gyroy_label">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>0</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
    </widget>
    <widget class="PlotWidget" name="gyroz" native="true">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>20</y>
       <width>221</width>
       <height>151</height>
      </rect>
     </property>
     <widget class="QLabel" name="gyroz_label">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>0</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>2</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Gyro_X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>2</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Gyro_Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>2</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Gyro_Z</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>200</y>
      <width>261</width>
      <height>156</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="scan_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>5</y>
       <width>261</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Scan</string>
     </property>
    </widget>
    <widget class="QComboBox" name="serial_comboBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>261</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="speed_comboBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>171</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="open_btn">
     <property name="geometry">
      <rect>
       <x>165</x>
       <y>60</y>
       <width>96</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="set_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>71</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stop_btn">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>120</y>
       <width>71</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="filename">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>95</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="start_btn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>71</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cali_btn">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>71</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Cali</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rec_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>71</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Rec</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stop_rec_btn">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>71</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>431</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="accelx">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>00.00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="accely">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>00.00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Z:</string>
     </property>
    </widget>
    <widget class="QLabel" name="accelz">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>00.00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Acceleration</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>431</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="magx">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>00.00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="magy">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>00.00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Z:</string>
     </property>
    </widget>
    <widget class="QLabel" name="magz">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>00.00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>3</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Magnet</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="status_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="threshold_in">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="duration">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>300</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Threshold</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Duration</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>425</x>
      <y>300</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>s</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>295</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>degree/s</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_btn">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>360</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CLOSE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gravity">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>360</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Gravity</string>
    </property>
   </widget>
   <widget class="QLabel" name="gravity_label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>330</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>degree/s</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="gyroxmax">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>degree/s</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="gyroymax">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>330</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>degree/s</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="gyrozmax">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>330</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>degree/s</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
